sci-calculators/bc-gh/bc-gh-4.0.0.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Gentoo Authors
```

Comment:
Strange, repoman should report this to you. It is 1999-2021

Reply:
> Strange, repoman should report this to you. It is 1999-2021

repoman/pkgcheck currently only care if date is outdated, but [devmanual](https://devmanual.gentoo.org/ebuild-writing/file-format/index.html#ebuild-header) wants the header.txt as-is

Reply:
I did update the new ebuild should I update the others?

Reply:
Does your own repoman complain about it? Please test before making unsure review comments.